awx/collections/ansible/tower/plugins/modules/tower_credential.py

```diff
@@ -0,0 +1,353 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# Copyright: (c) 2017, Wayne Witzel III <wayne@riotousliving.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Comment:
I'm pretty sure this is a no-no.

Reply:
As the AWX repo is Apache licensed, we can't add GPLv3 code in this way without changing the licensing around the repo (even if just to "Apache and GPLv3".)  That's a bigger discussion.

Given how these will live and be distributed in general as collections, this may be simpler to have them as a separate repo.